### PR TITLE
layout-stopwatch

### PR DESCRIPTION
**Pull request by svyatoslavPy**
- [DEMO LINK](https://svyatoslavPy.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,5 +10,12 @@
   </head>
   <body>
     <h1>Stop watch</h1>
+
+
```
Comment (volodymyr-soltys97) [inline src/index.html:14]:
```suggestion
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1 +1,95 @@
...
+$stopwatch-size: 80vmin;
+$stopwatch-border-color: #000;
+$green-color: #2c8000;
+$orange-color: #f6a603;
+$blue-color: #0700ff;
+$stopwatch-minutes-size: 3vmin;
+$stopwatch-minutes-length: 20vmin;
+$stopwatch-seconds-size: 1.5vmin;
+$stopwatch-seconds-length: 38vmin;
+$stopwatch-circle-size: 5vmin;
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:18]: Move variables to the separate file

File: src/styles/main.scss [hunk 3]
```diff
@@ -1 +1,95 @@
...
+
+    @keyframes move {
+      from {
+        transform: translate(-50%, -6%) rotate(0deg);
+      }
+
+      to {
+        transform: translate(-50%, -6%) rotate(360deg);
+      }
+    }
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:93]: You can use only one keyframes for the two arrows

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**